utils/file_utils.py

```diff
@@ -199,3 +200,31 @@ def chgrp(fname, grpname):
         raise RuntimeError('chown(%s) failed: %s' % (fname, str(e)))
 
 
+def from_url_or_local(target, destination):
```

Comment:
I had a utility routine like this somewhere else, wasn't it in utils?   FILEorSTRINGorURL or something like that.

Reply:
It was somewhere in repo build tools from 2015...